.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "important","updates"]
+assignees:
```

Comment:
you can remove this tag, this is optional and is set to octocat.

Reply:
done, but assignees section is auto generated